portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,20 +34,39 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-static ArrayList<String> myarraylist = new ArrayList<String>(); 
+//static ArrayList<String> myarraylist = new ArrayList<String>(); 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
+   Query query = new Query("Task").addSort("text", SortDirection.DESCENDING);
+   DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();    
+
+   PreparedQuery results = datastore.prepare(query);
+   
+   List<Task> task = new ArrayList<>();
+   for (Entity entity : results.asIterable()) {
+      long id = entity.getKey().getId();
+      String text = (String) entity.getProperty("text");
+
+      Task task2 = new Task(id, text);
+      task.add(task2);
+    }
+
     Gson gson = new Gson();
-    String json = gson.toJson(myarraylist);
     response.setContentType("application/json");
-    response.getWriter().println(json);
+    response.getWriter().println(gson.toJson(task));
  }
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
     String text = getParameter(request, "text-input", "");
     
-    myarraylist.add(text);
+    //myarraylist.add(text);
```

Comment:
Remove this line, or give relevant comment for the following lines (data store put).